styles.css

```diff
@@ -1,2 +1,54 @@
+:root {
+    --menu-bar-width: 60px;
+    --menu-bar-height: 10px;
+}
 
+#menu {
+    background-color: #D84282;
+    position: relative;
+    width: var(--menu-bar-width);
+    height: var(--menu-bar-height);
+}
 
+#menu::before,
+#menu::after {
+    background-color: #D84282;
+    display: block;
+    content: "";
+    width: var(--menu-bar-width);
+    height: var(--menu-bar-height);
+    position: absolute;
+    transition: all .2s ease-in-out;
+}
+
+#menu::before {
+    left: 0;
+    top: -20px;
+}
+
+#menu::after {
+    left: 0;
+    top: 20px;
+}
+
+#container {
+    display: block;
+    border: 1px red;
+    width: 60px;
+    height: 10px;
+    padding-bottom: 20px;
+    padding-top: 20px;
```

Comment:
Als je op regel 11 margin: 100px; hebt ingesteld, hoef je hier in de container geen margin en padding te gebruiken